### PR TITLE
meshsync update documented

### PR DESCRIPTION
**Pull request by Azanul**
**Notes for Reviewers**

This PR documents updating Meshsync CRD's informer configuration using kubectl.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 1384b5dbe68468819d5b7549e87df53caa3d988d at: https://646c9c92e1166818c49e74f4--meshery-docs-preview.netlify.app

File: docs/pages/concepts/architecture/meshsync.md [hunk 1]
```diff
@@ -28,7 +28,15 @@ The resources that are present inside the cluster are discovered efficiently wit
...
 ### Listening to events/changes on every component
 
-MeshSync implements several informers/listeners across each resource to listen to changes that occur to them. These are reflected at real time and are updated in their respective areas.
+The informer in MeshSync actively listens to changes in resources and updates them in real time based on the provided informer 
+configuration in the CRD.
 
 ## MeshSync FAQs
 
+### How to reconfigure Meshsync informer?  
+Meshsync is managed by [Operator]({{site.baseurl}}/concepts/architecture/operator) which watches for changes on Meshsync CRD for
```
Comment (leecalcote) [inline docs/pages/concepts/architecture/meshsync.md:37]:
```suggestion
MeshSync is managed by [Meshery Operator]({{site.baseurl}}/concepts/architecture/operator), which watches for changes on MeshSync CRD for 
```

**Comment by l5io** [comment]
🚀 Preview for commit 1dc028604647fedfec43703827d36b76e13dbad5 at: https://646d38227c01c8062aeb63ba--meshery-docs-preview.netlify.app

File: docs/pages/concepts/architecture/meshsync.md [hunk 2]
```diff
@@ -28,7 +28,15 @@ The resources that are present inside the cluster are discovered efficiently wit
...
 
 ## MeshSync FAQs
 
+### How to reconfigure Meshsync informer?  
+MeshSync is managed by [Meshery Operator]({{site.baseurl}}/concepts/architecture/operator), which watches for changes on MeshSync CRD for 
+changes and updates the deployed resource accordingly. You can update the CRD using kubectl through the following steps
+- Download the CRD with `kubectl get crd meshsyncs.meshery.layer5.io -o yaml > meshsync.yaml`
+- Open the downloaded file and edit the field `informer_config` to blacklist all the types of resources that you don't want updates from.
+- Apply the new definition with `kubectl apply -f meshsync.yaml` 
+
```
Comment (leecalcote) [inline docs/pages/concepts/architecture/meshsync.md:42]:
```suggestion
### How to configure MeshSync's resource discovery behavior: Can specific, "uninteresting" resources be blacklisted?  

MeshSync is managed by [Meshery Operator]({{site.baseurl}}/concepts/architecture/operator), which watches for changes on the `meshsync` CRD for changes and updates the deployed MeshSync instance accordingly. You can blacklist specifi Kubernetes resources from being discovered and watched by MeshSync. In order to identify the list of one or more resources for MeshSync to ignore, update the `meshsync` CRD using kubectl:

- Download the CRD with `kubectl get crd meshsyncs.meshery.layer5.io -o yaml > meshsync.yaml`
- Open the downloaded file and edit the field `informer_config` to blacklist all the types of resources that you don't want updates from.
- Apply the new definition with `kubectl apply -f meshsync.yaml` 

```

This FAQ needs a specific example that includes multiple resource names in the blacklist.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/concepts/architecture/meshsync.md (hunk 2 above)
Comment (Azanul) [reply docs/pages/concepts/architecture/meshsync.md:42]: suggestion has a "specifi" instead of "specific"

Comment (leecalcote) [reply docs/pages/concepts/architecture/meshsync.md:42]: Good eye. I've corrected the misspelling. 👍

**Pull request merged**